pennylane/templates/subroutines/phase_adder.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         self, k, x_wires, mod=None, work_wire=None, id=None
     ):  # pylint: disable=too-many-arguments
 
+        work_wire = qml.wires.Wires(work_wire)
```

Comment:
Might be worth adding, since people are used to being able to pass an integer for wires if there is only one wire. We could even update the docstring type hint to be `(Sequence[int] or int)`.

Reply:
I updated the type description in the docstring to reflect that `work_wire` might be an integer